app/schemas/schemas.coffee

```diff
@@ -65,15 +65,6 @@ me.extendBasicProperties = (schema, linkFragment) ->
   schema.properties = {} unless schema.properties?
   _.extend(schema.properties, basicProps(linkFragment))
 
-# Basic props for new schemas using json draft 07 
-basicPropsNew = (linkFragment) ->
```

Comment:
These new properties were created because ajv didnt support `format: hidden`. But now added support for unknown formats in ajv, hence these are not required.

Reply:
That's awesome.